test/test_misc_utils.py

```diff
@@ -1156,3 +1156,115 @@ def test_file_contents():
         assert file_contents("foo.bin", binary=True) == b'\x48\x65\x6c\x6c\x6f\x21\x0a'
         assert file_contents("foo.bin", binary=False) == b'\x48\x65\x6c\x6c\x6f\x21\x0a'.decode('utf-8')
         assert file_contents("foo.bin", binary=False) == 'Hello!\n'
+
+
+def make_counter(start=0):
```

Comment:
Now that we're in dcicutils, we might want to move this function to `misc_utils` or `qa_utils` so it's available to other things.